<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrorMessageDialog</class>
 <widget class="QDialog" name="ErrorMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Error Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblErrorMessage">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="lstErrorMessages"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="cmdShowHideDetails">
       <property name="text">
        <string>Show &amp;Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCopyToClipboard">
       <property name="text">
        <string>&amp;Copy to clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdSubmitToSupport">
       <property name="text">
        <string>&amp;Submit to support</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdOk">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cmdOk</sender>
   <signal>clicked()</signal>
   <receiver>ErrorMessageDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>596</x>
     <y>412</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>394</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
